data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh

```diff
@@ -15,9 +15,10 @@ while [ "$(oc get bmh -n openshift-machine-api -o name | wc -l)" -lt 1  ]; do
     sleep 20
 done
 
-while [ "$(oc get bmh -n openshift-machine-api -l installer.openshift.io/role=control-plane -o json | jq '.items[].status.provisioning.state' | grep -v provisioned -c)" -gt 0  ]; do
+N=$(oc get bmh -n openshift-machine-api -l installer.openshift.io/role=control-plane -o json | jq '.items[].status.provisioning.state' | wc -l)
```

Comment:
Probably don't need all those pipes if all we want to do is grab the number of control plane hosts